features/support/env.rb

```diff
@@ -57,13 +57,7 @@
 # Licensify admin doesn't have favicon.ico so block requests to prevent errors
 proxy.blacklist(/^https:\/\/licensify-admin(.*)\.publishing\.service\.gov\.uk\/favicon\.ico$/i, 200)
 
-chromedriver_from_path = File.which("chromedriver")
```

Comment:
Heads up we should keep this stuff, though we needed to remove it for local development.

Reply:
Oops - removed :-).